.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: ruby
+rvm:
+- 2.2.4
+before_install: gem update --remote bundler
+install:
+- bundle install --retry=3
+script:
+- bundle exec rspec
+- bundle exec rubocop --config .rubocop.yml
+deploy:
+  provider: heroku
+  run:
+  "rake db:migrate"
```

Comment:
I believe "rake db:migrate" should be right after run:  or indented with a dash like bundle exec rspec.